src/Libraries/CoreNodeModelsWpf/Charts/BarChartNodeModel.cs

```diff
@@ -60,7 +56,7 @@ public class BarChartNodeModel : NodeModel
         /// <summary>
         /// Bar chart color values.
         /// </summary>
-        public List<SolidColorBrush> Colors { get; set; }
+        public List<Color> Colors { get; set; }
```

Comment:
Since this is public - ideally we would obsolete the old property, and redirect it to to use and update the new one.

Reply:
Make sense since this was officially added in 2.18

Reply:
in addition @zeusongit @QilongTang - if this is going into 2.19 then we'll need to keep the old live charts binaries in the `legacy_remove_me` folder so they get copied.

If this is only going into a 3.0 release then we can forget the API break, and not include the binaries. I'm not sure that is an option though...? @jasonstratton @pinzart90 ?

Reply:
Maybe we can RC branch 2.19 early and make this a Dynamo 3.0 change, the downside is that chart nodes will not be tested in test fest early because that event is targeting 2.19 NET6 build.